ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/remote-worker/cucushift-installer-rehearse-vsphere-ipi-remote-worker-workflow.yaml

```diff
@@ -0,0 +1,11 @@
+workflow:
+  as: cucushift-installer-rehearse-vsphere-ipi-remote-worker
+  steps:
+    pre:
+    - chain: cucushift-installer-rehearse-vsphere-ipi-remote-worker-provision
+    post:
+    - chain: cucushift-installer-rehearse-vsphere-ipi-deprovision
+    env:
+      SIZE_VARIANT: "compact"
```

Comment:
@jianlinliu here's how I use compact.

Reply:
And here the install config https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/36786/rehearse-36786-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-vsphere-ipi-remote-worker/1635163112279642112/artifacts/installer-rehearse-vsphere-ipi-remote-worker/ipi-install-install/artifacts/install-config.yaml

Reply:
Ah, I missed that, how about moving it to provision chain yaml file?

Reply:
Moved to cucushift-installer-rehearse-vsphere-ipi-remote-worker-provision-chain